### PR TITLE
DCD-380: Add support for M5 instance types

### PR DESCRIPTION
**Pull request by nanux**

### CODE REVIEW

File: templates/quickstart-postgres-for-atlassian-services.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@
 # DBAllocatedStorageEncrypted
 # DBStorageType
 AWSTemplateFormatVersion: 2010-09-09
-Description: RDS Postgres Database for use in Atlassian Standard Infrastructure
+Description: RDS Postgres Database for use in Atlassian Standard Infrastructure (qs-1aj6s44e4)
```
Comment (bpar476) [inline templates/quickstart-postgres-for-atlassian-services.yaml]: What's `(qs-1aj6s44e4)` in the description for?

File: templates/quickstart-vpc-for-atlassian-services.yaml [hunk 2]
```diff
@@ -8,13 +8,11 @@ Metadata:
           default: ' VPC Network Configuration'
         Parameters:
           - AccessCIDR
-          - AssociatePublicIpAddress
           - AvailabilityZones
           - PrivateSubnet1CIDR
           - PrivateSubnet2CIDR
           - PublicSubnet1CIDR
           - PublicSubnet2CIDR
-          - SSLCertificateName
```
Comment (bpar476) [inline templates/quickstart-vpc-for-atlassian-services.yaml:17]: Why did we need to remove these parameters?

File: templates/quickstart-postgres-for-atlassian-services.yaml (hunk 1 above)
Comment (nanux) [reply templates/quickstart-postgres-for-atlassian-services.yaml]: They were added by Tony in upstream. I have just synced Postgres template and didn't sync Aurora. I will fix it.

Updated version:
https://github.com/aws-quickstart/quickstart-atlassian-services/blob/develop/templates/quickstart-postgres-for-atlassian-services.yaml

Documentation:
https://aws-quickstart.github.io/qs-ids.html
(they are generated by AWS)

Comment (nanux) [reply templates/quickstart-postgres-for-atlassian-services.yaml]: I have synced `develop` with upstream aaand they are gone (from the PR).

File: templates/quickstart-vpc-for-atlassian-services.yaml (hunk 2 above)
Comment (nanux) [reply templates/quickstart-vpc-for-atlassian-services.yaml:17]: They are not used in the template at all. They are some leftovers.
They are also not mentioned in the guide as well - https://aws-quickstart.s3.amazonaws.com/quickstart-atlassian-services/doc/atlassian-standard-infrastructure-on-the-aws-cloud.pdf

**Review by bpar476 — APPROVED** [review]

**Review by theghostwhoforks — COMMENTED** [review]
LGTM

**Review by theghostwhoforks — APPROVED** [review]

**Pull request merged**